thyrel-web/src/components/GetRandomValueFromArray.tsx

```diff
@@ -0,0 +1,3 @@
+export default function GetRandomValueFromArray<T>(possibilities: T[]): T {
```

Comment:
```suggestion
export default function getRandomValueFromArray<T>(possibilities: T[]): T {
```
(faut changer partout aussi)